.github/workflows/gen-release-pr.yml

```diff
@@ -0,0 +1,24 @@
+name: Generate release pull request
+
+on:
+  workflow_dispatch:
+
+jobs:
+  generate-pr:
+    runs-on: ubuntu-latest
+    container: codetent/crypta-dev:1.0
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Batch and merge
+      run: |
+        changie batch auto
+        changie merge
+        echo RELEASE_VERSION=$(changie latest) >> $GITHUB_ENV
+    - name: Create Pull Request
```

Comment:
I saw this in the changie repository. I am wondering why this does not need a github token?

Reply:
I guess this is possible using the integrated workflow token. Not sure if the correct rights are set for it. But let's test....